src/ss_log_config.json

```diff
@@ -21,7 +21,7 @@
             "class": "logging.handlers.RotatingFileHandler",
             "level": "DEBUG",
             "formatter": "basic_format",
-            "filename": "../logs/ss_debug.log",
+            "filename": "/home/sar/catkin_ws/src/sar_social_stories/logs/ss_debug.log",
```

Comment:
Please don't commit your personal config file to the repository. If we decide to switch to using an absolute path, the example config in the repository should have an example path (e.g., "/path/to/logs/ss_debug.log"), not the path on your personal machine.